ci/prow/make_config.go

```diff
@@ -264,6 +265,51 @@ const (
 	dashboardGroupTemplate = "testgrid_dashboardgroup.yaml"
 )
 
+// Generate cron string based on job type, offset generated from jobname
+// instead of assign random value to ensure consistency among runs
+func generateCron(jobType, jobName string) string {
+	getUTCtime := func(i int) int { return i + 7 }
+
+	// sums the ascii valus of all letters in a jobname,
+	// this value is used for deriving offset after hour
+	var sum float64
+	for _, c := range jobName + jobType {
+		sum += float64(c)
+	}
+	minutesOffset := int(math.Mod(sum, 60))
+	everyHourCron := fmt.Sprintf("%d * * * *", minutesOffset)
+	everyOtherHourCron := fmt.Sprintf("%d */2 * * *", minutesOffset)
+	everydayCron := fmt.Sprintf("%d %%d * * *", minutesOffset)    // hour
```

Comment:
This looks nice!
Just one question, what does `%%d` mean here?

Reply:
It's the literal formatting for the real value. See line 304 for example.